torch/lib/THD/CMakeLists.txt

```diff
@@ -155,10 +160,8 @@ IF(MPI_FOUND)
   ENDIF()
 ENDIF()
 
-IF(GLOO_FOUND)
-  INCLUDE_DIRECTORIES(${GLOO_INCLUDE_DIR})
-  FILE(APPEND "${CMAKE_INSTALL_PREFIX}/THD_deps.txt" "${Gloo_LIBRARY};")
-  FILE(APPEND "${CMAKE_INSTALL_PREFIX}/THD_deps.txt" "${Gloo_NATIVE_LIBRARY};")
+IF(USE_GLOO AND USE_CUDA)
```

Comment:
GLOO is used for CPU-only backend as well right?
isn't it `IF(USE_GLOO) ADD_DEPENDENCIES(THD gloo)`